SpatialGDK/Source/Private/Interop/SpatialOutputDevice.cpp

```diff
@@ -25,7 +28,7 @@ void FSpatialOutputDevice::Serialize(const TCHAR* InData, ELogVerbosity::Type Ve
 		return;
 	}
 
-	if (Connection->IsConnected())
+	if (bLogToSpatial && Connection->IsConnected())
```

Comment:
Specific overriding log type still coming or did you decide against it?

Reply:
Decided against it for now. Can add later.